data/client/king_phisher/client_config.json

```diff
@@ -68,5 +69,5 @@
   "ssh_server": "localhost:22",
   "ssh_username": "",
   "text_font": "monospace 11",
-  "text_source.theme": "cobalt"
+  "text_source.theme": "cobalt",
```

Comment:
I think this is technically invalid JSON. You can't have a trailing comma, but King Phisher makes a best effort to address this type of issue to be user friendly. I'd prefer to keep the files that ship with the source conforming to the spec though.